reference/docs-conceptual/SecretManagement/get-started/using-secretstore.md

```diff
@@ -0,0 +1,82 @@
+---
+description: How to install and use the SecretManagement and SecretStore modules
+ms.date: 05/24/2022
+title: Get started with the SecretStore module
+---
+# Get started with the SecretStore module
+
+The **SecretManagement** and **SecretStore** modules are available from the PowerShell Gallery and
+can be installed using **PowerShellGet** commands.
+
+```powershell
+Install-Module Microsoft.PowerShell.SecretManagement
+Install-Module Microsoft.PowerShell.SecretStore
+```
+
+Once you have installed the modules you can load the modules and begin using or creating new secrets.
+
+```powershell
+Import-Module Microsoft.PowerShell.SecretManagement
+Import-Module Microsoft.PowerShell.SecretStore
+```
+
+## Create a vault and add a secret
+
+First you must register the vault. The **Name** parameter is a friendly name and can be anything you
+choose.
+
+```powershell
+Register-SecretVault -Name SecretStore -ModuleName Microsoft.PowerShell.SecretStore -DefaultVault
+```
+
+The **DefaultVault** parameter makes this the default vault.
```

Comment:
If I have no created/registered vaults, is the first vault created automatically the default? If not, how is the behavior different if I don't have a default vault and should we add a note on this here?

Reply:
Unknown. Leave as is for now. Needs more investigation and discussion with Dev/PM.

Reply:
The 'default' vault is a registered vault that is used with the SecretManagement cmdlets when no vault is specified from parameter arguments.  The first registered vault is always made 'default', but any vault can be made 'default' via the 'Set-SecretVaultDefault' cmdlet.

Reply:
This is a good note for the cmdlet reference.